src/crypto/public_key/elliptic/oid_curves.js

```diff
@@ -57,7 +57,8 @@ const curves = {
     node: nodeCurves[enums.curve.nistP256],
     web: webCurves[enums.curve.nistP256],
     payloadSize: 32,
-    sharedSize: 256
+    sharedSize: 256,
+    wireFormatLeadingByte: 0x04
```

Comment:
Perhaps we can reuse this param in generation/serialization too? (Not super critical ofc)